pom.xml

```diff
@@ -160,9 +160,6 @@
                 -->
                 <plugin>
                     <artifactId>maven-javadoc-plugin</artifactId>
-                    <configuration>
-                        <legacyMode>true</legacyMode>
-                    </configuration>
                 </plugin>
                 <plugin>
```

Comment:
Do we still use the old plugin?

Reply:
After this PR, all the modules are javadoc'd as JPMS modules so we don't need `legacyMode` anymore. We do still use the javadoc plugin though if that's what you're asking?

Reply:
No, the `<groupId>io.github.dmlloyd.module-info</groupId>` :)

Reply:
Ah, no it should be removed now.

Reply:
Fixed